src/components/app-configure/auth/WebWallet.vue

```diff
@@ -78,14 +78,17 @@ function hasSameValuesInStore() {
 
 async function handleSave() {
   try {
-    const { auth } = app
-    auth.wallet.websiteDomain = walletWebsiteDomain.value
-    auth.wallet.walletType = hasUIMode.value ? WalletMode.UI : WalletMode.NoUI
-    auth.wallet.selectedTheme = selectedTheme.value.value
+    const { auth } = { ...app }
```

Comment:
why are spreading before de-structuring?

Reply:
Yeah actually not required. Removed [here](https://github.com/arcana-network/developer-dashboard/pull/135/commits/7ace58432fb05f1afdc58d173ea1be8c9918b9ab)